woocommerce/woocommerce
# Style new wizard 💅

**Steps to test:**
1. Check out this branch
2. `grunt watch`
3. Go to these pages:
wp-admin/index.php?page=wc-setup
wp-admin/index.php?page=wc-setup&step=payment
wp-admin/index.php?page=wc-setup&step=shipping
wp-admin/index.php?page=wc-setup&step=extras
wp-admin/index.php?page=wc-setup&step=activate
wp-admin/index.php?page=wc-setup&step=next_steps

**Current status:**
✅ all pages are styled except the last one
o Styling the last page - 'Next Steps'
